src/generator/passes.jl

```diff
@@ -32,7 +32,15 @@ function (x::CollectTopLevelNode)(dag::ExprDAG, options::Dict)
         header_name = normpath(spelling(tu_cursor))
         @info "Processing header: $header_name"
         for cursor in children(tu_cursor)
-            file_name = normpath(get_filename(cursor))
+            file_name = get_filename(cursor)
+
+            # Ignore items where no source file is available.
+            # e.g. built-in defines like `__llvn__` and `__clang__`.
```

Comment:
I guess these symbols should be defined in the header files of the compiler's include directory.

Reply:
No, these symbols don't come from header files, they are hard-coded in the source code of the compiler line this:
```c++
  // Compiler version introspection macros.
  Builder.defineMacro("__llvm__");  // LLVM Backend
  Builder.defineMacro("__clang__"); // Clang Frontend
```
See  https://github.com/llvm/llvm-project/blob/8699efba6dc8d905d144727eb10ff6782376bb87/clang/lib/Frontend/InitPreprocessor.cpp#L744-L746